upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -709,6 +682,17 @@ func (c *ApplyClusterCmd) Run() error {
 		// Avoid making changes on a dry-run
 		shouldPrecreateDNS = false
 
+	case TargetInventory:
+		// Utilized to build an api.Inventory
+		// This target was created because TargetDryRun outputs to STDOUT.  This allows for the rendering of
```

Comment:
Let's reuse dryrun target.  If we need to pass stdout & stderr streams, let's do that :-)